### PR TITLE
Consolidate calls to get atime and stime to use File#stat

### PR DESCRIPTION
**Pull request by melvinram**
Cuts two two file system calls to just one call.

@zendesk/zendesk-email @grosser @staugaard

### CODE REVIEW

**Comment by grosser** [comment]
:+1:

File: lib/adrian/file_item.rb [hunk 1]
```diff
@@ -28,16 +28,18 @@ def move(destination)
       @value = destination_path
     end
 
-    def atime
-      File.atime(path).utc
+    def stat
+      @stat = File.stat(path)
```
Comment (melvinram) [inline lib/adrian/file_item.rb]: Just noticed that I didn't put in the memorization that I had intended.

**Comment by ggrossman** [comment]
You could change the

```ruby
created_at
updated_at
```

at the top to just

```ruby
stat
```

**Comment by ggrossman** [comment]
:+1:

**Comment by staugaard** [comment]
:+1:

**Pull request merged**